app/api/feedbacks.py

```diff
@@ -27,7 +27,7 @@ def before_post(self, args, kwargs, data):
         :return:
         """
         require_relationship(['event', 'user'], data)
-        if not has_access('is_user_itself', id=int(data['user'])):
+        if not has_access('is_user_itself', user_id=int(data['user'])):
```

Comment:
After solving the actual bug related to deletion of feedback, i was getting error in creating a feedback. So i had to edit it here also.

Reply:
Yeah, so this is the problem with kwargs, they are too dynamic. But I am not sure how it can be solved project wise. Ideally, function arguments must be concrete, but here, using kwargs, anyone can pass any named argument.

Can you search the entire project with `has_access('is_user_itself'` and check if all instances use `user_id` key

IMO, best way to mitigate it is to use the concrete function instead. 

CC @niranjan94 @SaptakS @shubham-padia

Reply:
No, a few of them are using ```id```. For eg. Event, I guess this is the cause for HTTP 500 in creating  event also as mentioned in #4654

Reply:
Should i correct them all in this PR only ?

Reply:
Yeah I think similar issues should be fixed in this PR